sdk/mysql/Azure.ResourceManager.MySql/src/MySqlFlexibleServers/Custom/RestOperations/BackupAndExportRestOperations.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using Azure.Core;
+using Azure.ResourceManager.MySql.FlexibleServers.Models;
+using Azure.ResourceManager.MySql.MySqlFlexibleServers;
+
+namespace Azure.ResourceManager.MySql.FlexibleServers
+{
+    internal partial class BackupAndExportRestOperations
+    {
+        internal HttpMessage CreateCreateRequest(string subscriptionId, string resourceGroupName, string serverName, MySqlFlexibleServerBackupAndExportContent content)
+        {
+            var context = new RequestContext();
```

Comment:
Add TODO to be able to apply policy for operation

Reply:
Added